src/ontology/hp.Makefile

```diff
@@ -636,3 +636,77 @@ $(TMPDIR)/hp-%-merged.owl: hp-base.owl tmp/%.owl
 
 mappings: 
 	$(MAKE_FAST) ../mappings/hp-snomed.lexmatch.sssom.tsv
+
+.PHONY: diff
+diff: diff-release-base diff-main-branch-base
+
+tmp/hp-edit-pr.obo: hp-edit.owl
```

Comment:
why is this called `-pr`? The make goal is simply `tmp/hp-edit.obo`, nothing PR specific about this.